README.md

```diff
@@ -88,6 +88,8 @@ These containers are set to persist between sessions via the `persistent` parame
 make bufinstall && make buf 
 # Install dependencies
 make setup
+# Install lua files
+make install-lua-files
```

Comment:
Should we do this step after `make build` to match what we do in the appimage workflow (https://github.com/EshaMaharishi/viam-cartographer/blob/bbeb5194514f988d15b14bc336578e370ec48154/.github/workflows/appimage.yml#L64)?

Or alternatively should we make the appimage workflow follow the order here?

Reply:
good idea - will align it with how the appimage workflow looks